pylib/anki/decks.py

```diff
@@ -283,24 +282,44 @@ def _canDragAndDrop(self, draggedDeckName: str, ontoDeckName: str) -> bool:
             return True
 
     def _isParent(self, parentDeckName: str, childDeckName: str) -> Any:
-        return self._path(childDeckName) == self._path(parentDeckName) + [
-            self._basename(childDeckName)
+        return self.path(childDeckName) == self.path(parentDeckName) + [
+            self.basename(childDeckName)
         ]
 
     def _isAncestor(self, ancestorDeckName: str, descendantDeckName: str) -> Any:
-        ancestorPath = self._path(ancestorDeckName)
-        return ancestorPath == self._path(descendantDeckName)[0 : len(ancestorPath)]
+        ancestorPath = self.path(ancestorDeckName)
+        return ancestorPath == self.path(descendantDeckName)[0 : len(ancestorPath)]
 
-    def _path(self, name: str) -> Any:
+    @staticmethod
+    def path(name: str) -> Any:
         return name.split("::")
 
-    def _basename(self, name: str) -> Any:
-        return self._path(name)[-1]
+    _path = path
```

Comment:
What is the rationality behind having both `path` and `_path`

Reply:
preserving compatibility with code using the old style